src/database/schemas/address.schema.ts

```diff
@@ -0,0 +1,59 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document } from 'mongoose';
+import { CountryEnum } from '../enums/country.enum';
+
+export type AddressType = Address & Document;
+
+@Schema({ _id: false })
+export class Address {
+    @Prop({
+        type: String,
+        trim: true,
+        lowercase: true,
+        maxLength: 256,
```

Comment:
issue(blocking): This is way too long. Reduce to half of it. This should be big enough.
thought(non-blocking): We could also block any kind of special characters like "!=§?" and so on

Reply:
We are not really using this schema actively and i dont think we will in the near future. Maybe we should do optimizations at some other point.